services/views.py

```diff
@@ -358,9 +359,16 @@ def activate_mumble(request):
     logger.debug("activate_mumble called by user %s" % request.user)
     authinfo = AuthServicesInfoManager.get_auth_service_info(request.user)
     character = EveManager.get_character_by_id(authinfo.main_char_id)
+    # Blue members should have alliance ticker (if in alliance)
```

Comment:
this logic here should be moved into the block below 361/369 (only executed on blues)